doc/modules/preprocessing.rst

```diff
@@ -105,6 +88,8 @@ It is possible to disable either centering or scaling by either
 passing ``with_mean=False`` or ``with_std=False`` to the constructor
 of :class:`StandardScaler`.
 
+See also :func:`scale`.
+
```

Comment:
I would prefer to see the `sklearn.preprocessing.scale` function only in the `See Also` section of `sklearn.preprocessing.StandardScaler` reference.

Reply:
Do you mean [here](https://scikit-learn.org/dev/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler)?
They already have it in that documentation, so I can just delete this piece of code.

Reply:
Yep, right there.